configs/components/facter.rb

```diff
@@ -197,6 +203,12 @@
 
   pkg.install_file ".gemspec", "#{settings[:gem_home]}/specifications/#{pkg.get_name}.gemspec"
 
+  if platform.is_windows?
+    pkg.install_file "../facter.bat", "#{settings[:bindir]}/facter.bat"
+    pkg.install_file "../facter_interactive.bat", "#{settings[:bindir]}/facter_interactive.bat"
+    pkg.install_file "../run_facter_interactive.bat", "#{settings[:bindir]}/run_facter_interactive.bat"
```

Comment:
Why? That's where the resources were (previously). Because I added them as a sources (at the top of this file), they were a directory above the current working directory. I decided this wasn't how I wanted to deal with these files though.